__tests__/app.test.js

```diff
@@ -52,39 +52,110 @@ describe("GET /api/articles/:article_id", () => {
+  });
+  test("404: correct data type but does not exist", () => {
+    return request(app)
+      .get("/api/articles/7777777")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("404: Not Found");
+      });
+  });
 });
 
 describe("Get /api/users", () => {
-    test("returns status 200 when successful", () => {
-        return request(app)
-        .get("/api/users")
-        .expect(200)
-        .then(({body}) => {
-            expect(body.users.length).toBe(4)
-            expect(body.users).toBeInstanceOf(Array);
-            expect(body.users.forEach((users) => {
-                expect(users).toEqual(expect.objectContaining({
-                    username: expect.any(String),
-                    name: expect.any(String),
-                    avatar_url: expect.any(String),
-                }))
-            }))
-        })
-    })
-})
+  test("returns status 200 when successful", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
+        expect(body.users).toBeInstanceOf(Array);
+        expect(
+          body.users.forEach((users) => {
+            expect(users).toEqual(
+              expect.objectContaining({
+                username: expect.any(String),
+                name: expect.any(String),
+                avatar_url: expect.any(String),
+              })
+            );
+          })
+        );
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("201: responds with updated article", () => {
+    const updateAdd = {
+      inc_votes: 3,
+    };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(updateAdd)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 3,
+        });
+      });
+  });
+  test("201: responds with updated article", () => {
+    const updateMinus = {
+      inc_votes: -3,
+    };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(updateMinus)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: -3,
+        });
+      });
+  });
+  test("404: correct data type but does not exist", () => {
+    const test = {
+      inc_votes : 7
+    }
+    return request(app)
+      .patch("/api/articles/7777")
+      .send(test)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("404: Not Found");
+      });
+  });
```

Comment:
Please, add a test for when we are making the request to an id that is not a number 
i.e. "/api/articles/just_a_string"